.ci/Dockerfile

```diff
@@ -92,6 +92,12 @@ RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/s
 
 RUN helm plugin install https://github.com/quintush/helm-unittest --version 0.2.8
 
+ENV VAULT_VERSION=1.12.3
```

Comment:
Do we also need to add this to the buildkite agent image?

Reply:
No, this is handled by an environment hook in the Buildkite agent itself, like vault login, whether it's a gcp or k8s agent.